src/Pages/Category/category-card.jsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom";
+
+function CategoryCard(props) {
+  return (
+    <>
+      <Link to="/quiz" className="color-white">
+        <div class="card m-1">
+          <div class="card-header card-overlay">
+            <img
+              src="https://www.hubspot.com/hubfs/marketer%20building%20a%20survey%20through%20the%20best%20online%20quiz%20makers.jpg"
+              alt="..."
```

Comment:
Proper Alt message will be better

Reply:
Thank you for pointing this will fix it.